lib/projectassignment/errors.go

```diff
@@ -10,37 +10,68 @@ type ProjectsMissingError struct {
 	projectsMissing []string
 }
 
-func NewProjectsMissingErroror(projectsMissing []string) error {
+func NewProjectsMissingError(projectsMissing []string) error {
 	return &ProjectsMissingError{projectsMissing: projectsMissing}
 }
 
 func (e *ProjectsMissingError) Error() string {
 	var errorStr string
 	if len(e.projectsMissing) > 1 {
-		errorStr = fmt.Sprintf("You cannot modify projects for this object because these projects did not exist: %q", e.projectsMissing)
+		errorStr = fmt.Sprintf("Projects for this object cannot be modified because these projects did not exist: %q", e.projectsMissing)
```

Comment:
1. Changing "did" to "do" was apparently missed so that should be fixed for consistency.
2. Consider also deleting "for this object" as that context seems clear.
```suggestion
		errorStr = fmt.Sprintf("Projects cannot be modified because these projects do not exist: %q", e.projectsMissing)
```